--- /usr/share/initramfs-tools/scripts/zfs	2024-08-20 15:14:18.170417535 +0200
+++ /usr/share/initramfs-tools/scripts/zfs	2024-08-20 15:17:11.195822061 +0200
@@ -228,6 +228,12 @@
 	[ "$quiet" != "y" ] && zfs_log_begin_msg \
 		"Importing pool '${pool}' using defaults"
 
+        # Setup Loopback Devices first
+        if [ -x "/usr/sbin/looptab" ]
+        then
+            /usr/sbin/looptab
+        fi
+
 	ZFS_CMD="${ZPOOL} import -N ${ZPOOL_FORCE} ${ZPOOL_IMPORT_OPTS}"
 	ZFS_STDERR="$($ZFS_CMD "$pool" 2>&1)"
 	ZFS_ERROR="$?"
